src/pybind/mgr/dashboard_v2/frontend/src/app/core/auth/login/login.component.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { LoginComponent } from './login.component';
+import { FormsModule } from '@angular/forms';
+import { SharedModule } from '../../../shared/shared.module';
+import { RouterTestingModule } from '@angular/router/testing';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { ToastModule } from 'ng2-toastr';
+
+describe('LoginComponent', () => {
+  let component: LoginComponent;
+  let fixture: ComponentFixture<LoginComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      imports: [
+        FormsModule,
+        SharedModule,
+        RouterTestingModule,
+        HttpClientTestingModule,
+        ToastModule.forRoot()
+      ],
+      declarations: [
+        LoginComponent
+      ]
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(LoginComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
```

Comment:
Can be replaced by using the ComponentFixtureAutoDetect provider.

Reply:
This was generated by the "ng generate component" so I will not change it at the moment, to guarantee that we have the same structure on all tests.

We can change it later when we stop to focus on the frontend unit tests.